### PR TITLE
Add minor changes to be compatible with platform project

### PR DESCRIPTION
**Pull request by franck-fourel**

### CODE REVIEW

File: artifact/artifact_go.sh [hunk 1]
```diff
@@ -70,27 +73,37 @@ main() {
...
+        APP="${TRAVIS_REPO_SLUG#*/}-${PARAM_APP_NAME}"
+        DOCKER_CMD="-f Dockerfile.${PARAM_APP_NAME}"
+    else
         APP="${TRAVIS_REPO_SLUG#*/}"
-        APP_DIR="${ARTIFACT_DIR}/${APP}"
-        APP_TAG="${APP}-${TRAVIS_TAG}"
-
-        rm -rf "${ARTIFACT_DIR}/" || { echo 'ERROR: Unable to delete artifact directory'; exit 1; }
-        mkdir -p "${APP_DIR}/" || { echo 'ERROR: Unable to create app directory'; exit 1; }
+    fi
```
Comment (BigFab) [inline artifact/artifact_go.sh:82]: we should move these lines at the top so we can echo "Working on $APP" even if no argument is given

File: artifact/artifact_go.sh [hunk 2]
```diff
@@ -43,9 +43,12 @@ triggerSecurityScan() {
 }
 
 main() {
+    # $1 name of the app
+    local PARAM_APP_NAME=${1:-} 
+    echo "Working on ${PARAM_APP_NAME}"
```
Comment (BigFab) [inline artifact/artifact_go.sh:48]:
```suggestion
    echo "Working on ${APP}"
```

**Review by BigFab — CHANGES_REQUESTED** [review]
Some request for changes inline.
I also need to test that on a "regular" Go project build to ensure the pipeline is not broken

File: artifact/Readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Artifact Tools
```
[the remark this answers is not in the document]
Comment (franck-fourel) [reply artifact/Readme.md]: done

File: artifact/artifact_go.sh (hunk 1 above)
Comment (franck-fourel) [reply artifact/artifact_go.sh]: done

**Review by BigFab — APPROVED** [review]
LGTM

**Pull request merged**